files/gitconfig

```diff
@@ -0,0 +1,5 @@
+[user]
+    name = Packit Service
+    email = user-cont-team+packit-service@redhat.com
```

Comment:
@TomasTomecek I've just noticed that the original conversions on `git.stg.centos.org` was done using `Packit <packit>`. Should I rather follow that pattern or using this is also good? Any preference?

Reply:
this is definitely better since it's a valid e-mail :)